client/packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -151,9 +151,15 @@ export const AutocompleteList = <T,>({
             },
           },
           listbox: {
-            style: {
+            sx: {
               minHeight: height ? `${height}` : 'auto',
               maxHeight: height ? `${height}` : 'auto',
+              '& .MuiTypography-root': {
+                maxWidth: 497,
```

Comment:
I think if possible it would be ideal that we don't set the maxWidth here, rather use up the remaining space left in the div. Not sure if this will work but e.g.

```ts
              {
                width: '100%',
                flex: '1',
                whiteSpace: 'nowrap',
                overflow: 'hidden',
                textOverflow: 'ellipsis',
              }
```

Reply:
If it doesn't work then static values the way

Reply:
I tried, doesn't work